lib/node/runner/api_block.go

```diff
@@ -179,7 +179,9 @@ func UnmarshalNodeItemResponse(d []byte) (itemType NodeItemDataType, b interface
 		err = unmarshal(&t)
 		b = t
 	case NodeItemTransaction:
-		b, err = transaction.NewTransactionFromJSON(d[len(sc.Bytes())+1:])
+		var t transaction.Transaction
+		err = unmarshal(&t)
+		b = t
```

Comment:
That looks like a mistake

Reply:
It is intended.

Reply:
Oh I see what you did! Yeah that makes sense then! 👍